### DISCUSSION TITLE
Firebase Security for vs-code-live

### DISCUSSION THREAD

**Discussion by jul-fls** [coder/code-server#3028] [Q&A]
Hello, i've setup all this system for a multi user use and recently google emailed me about security risks on my db and presents unauthorized rules that anyone car read and write on my db and i receive this email each day at the same time so i want to get rid of that so how can i secure my db, i've search on the firepad side and found a set of rules that is good for google side of things but does'nt work for vs-code-live because it requires authentification, so it does'nt work, can somebody explain me how to secure my db and still have vs-code-live in code-server functionning ?
Please help me.
Thank you very much !

**Comment by jsjoeio** [comment]
Hey @jul-fls - I think you're referring to [vscode-live](https://github.com/coderpair/vscode-live) which is a project maintained by the community. I suggest opening up your issue/discussion there as we don't oversee that project!

**Reply by jul-fls** [reply]
Ok thank you but when i click here 
![image](https://user-images.githubusercontent.com/50296610/113601912-3ec46b80-9642-11eb-8bab-1e6b8780f969.png) it sends me here to create a discussion that's what i did.
And i've seen the same issue than me also posted here https://github.com/cdr/code-server/discussions/2717 so i think here is the only place to ask that ('ve just seen that i didn't know the issue has been posted here when i've posted this discussion).

**Reply by jsjoeio** [reply]
Woah! Didn't realize they were linking to us. I will post an issue asking them to fix that. Thanks for bringing it up.

Does #2717 solve your issue then?

**Reply by jul-fls** [reply]
Not really because it says to use Firestore instead of Firebase Realtime Database but i don't know how to implement that in my case and there's no documentation for it so i have 2 possibilities, the first is that i wait here that somebody can help me, the 2nd is to ask in te other ask how can i implement it.

**Reply by jsjoeio** [reply]
Ah, bummer. Apologies, @jul-fls I"m not super familiar with Firebase. Maybe someone else will come by who knows more. Maybe you've already done this, but there may be some helpful Firebase videos on this channel Fireship: [see here](https://www.youtube.com/c/AngularFirebase/search?query=firebase)
